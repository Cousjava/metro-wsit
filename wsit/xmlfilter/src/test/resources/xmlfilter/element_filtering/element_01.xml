<!--

    Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.

    This program and the accompanying materials are made available under the
    terms of the Eclipse Distribution License v. 1.0, which is available at
    http://www.eclipse.org/org/documents/edl-v10.php.

    SPDX-License-Identifier: BSD-3-Clause

-->

<wsp:Policy 
    xmlns:wsp="http://schemas.xmlsoap.org/ws/2004/09/policy"
    xmlns:policy="http://java.sun.com/xml/ns/wsit/policy" 
    xmlns:wsss="http://schemas.sun.com/2006/03/wss/server"    
    xmlns:wssc="http://schemas.sun.com/2006/03/wss/client"    
    xmlns:scs="http://schemas.sun.com/ws/2006/05/sc/server"    
    xmlns:scc="http://schemas.sun.com/ws/2006/05/sc/client"    
    xmlns:trusts="http://schemas.sun.com/ws/2006/05/trust/server"  
    xmlns:trustc="http://schemas.sun.com/ws/2006/05/trust/client"    
    xmlns:ns0="http://com.foo/goo"
>
    <ns0:Assertion0/>
    
    <wsss:KeyStore></wsss:KeyStore>
    <wsss:TrustStore></wsss:TrustStore>
    <wsss:CallbackHandlerConfiguration></wsss:CallbackHandlerConfiguration>
    <wsss:ValidatorConfiguration></wsss:ValidatorConfiguration>
    <wsss:DisablePayloadBuffering></wsss:DisablePayloadBuffering>
    
    <wssc:KeyStore></wssc:KeyStore>
    <wssc:TrustStore></wssc:TrustStore>
    <wssc:CallbackHandlerConfiguration></wssc:CallbackHandlerConfiguration>
    <wssc:ValidatorConfiguration></wssc:ValidatorConfiguration>
    <wssc:DisablePayloadBuffering></wssc:DisablePayloadBuffering>
    
    <scs:SCConfiguration></scs:SCConfiguration>
    
    <scc:SCClientConfiguration></scc:SCClientConfiguration>
    
    <trusts:STSConfiguration></trusts:STSConfiguration>
    
    <trustc:PreconfiguredSTS></trustc:PreconfiguredSTS>
    
    <ns0:Assertion1 policy:visibility="private" a="value"/>
    <ns0:Assertion2 a1="value1" a2="value2">content</ns0:Assertion2>
    <ns0:Assertion3 policy:visibility="private">
        100
    </ns0:Assertion3>
    
</wsp:Policy>

