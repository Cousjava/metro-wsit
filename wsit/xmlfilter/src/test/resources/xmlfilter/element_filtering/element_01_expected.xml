<!--

    Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.

    This program and the accompanying materials are made available under the
    terms of the Eclipse Distribution License v. 1.0, which is available at
    http://www.eclipse.org/org/documents/edl-v10.php.

    SPDX-License-Identifier: BSD-3-Clause

-->

<wsp:Policy 
    xmlns:wsp="http://schemas.xmlsoap.org/ws/2004/09/policy"
    xmlns:policy="http://java.sun.com/xml/ns/wsit/policy" 
    xmlns:ns0="http://com.foo/goo"
>
    <ns0:Assertion0/>        
    <ns0:Assertion1 policy:visibility="private" a="value"/>
    <ns0:Assertion2 a1="value1" a2="value2">content</ns0:Assertion2>
    <ns0:Assertion3 policy:visibility="private">
        100
    </ns0:Assertion3>
    
</wsp:Policy>

